### ISSUE TITLE
Merge into upstream?

### ISSUE THREAD

**Issue by diamondman** [dsigma/dfu-util#3]
It seems that a lot of people use this fork when working with M4 devices. Do the primary maintainers consider your patch to be out of scope? What is required to get this in the main release?

**Comment by dsigma** [comment]
At the time I forked it I wasn't getting a response from the maintainers, and it was rather specific so I didn't think it was of much use. I would much prefer to have this upstream and just use the mainline version. It's probably just a matter of making a patch and them patching their version. I'll reach out to them and see if they are amenable to incorporating the patch.

**Comment by dsigma** [comment]
I wanted to mention, I looked at the latest code on sourceforge
https://sourceforge.net/p/dfu-util/dfu-util/ci/master/tree/
and chatted with the maintainer a little and it looks like the equivalent functionality is now available in the mainline branch. It's done with the ":leave" syntax on the --dfu-useaddress command. See
http://dfu-util.sourceforge.net/dfu-util.1.html
for details on how to use that.
I tested this on my hardware and it works as expected.
